server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/common/CICSTestUtils.java

```diff
@@ -62,7 +62,7 @@ public static String getTestString(
     instances.replaceAll(String.join("", Collections.nCopies(12, " "))::concat);
     List<String> compilerOptionsList =
         Arrays.stream(compilerOptions)
-            .map(compilerOption -> compilerOption = "       CBL CICS(\"" + compilerOption + "\")")
+            .map(compilerOption -> compilerOption = "       CBL CICS('" + compilerOption + "')")
```

Comment:
This should still work.
```suggestion
            .map(compilerOption -> compilerOption = "       CBL CICS(\"" + compilerOption + "\")")
```